client/odysseus/message/index.tsx

```diff
@@ -1,9 +1,12 @@
 import classnames from 'classnames';
-import { RefObject, useState } from 'react';
+import { lazy, RefObject, Suspense, useState } from 'react';
 import type { Message } from '../types';
 
 import './style.scss';
 
+// eslint-disable-next-line import/no-extraneous-dependencies
+const ReactMarkdown = lazy( () => import( 'react-markdown' ) );
```

Comment:
I wonder if we could use Calypso's `AsyncLoad` here. Calypso is not quite optimized to work with React's `lazy/import` and `AsyncLoad` is so broadly adopted already. 

We might want to migrate away to `lazy/import` at some point, but we might need to consider some under the hood optimizations first.

Reply:
Migrated - can you check out the implementation and let me know if it's ok?

Reply:
Works ok 👍 You could optionally remove the default placeholder since it's usually not something folks want in their UI.